forward_engineering/helpers/viewHelper.js

```diff
@@ -13,36 +13,48 @@ const { getOptions, getPrimaryKeyList } = require('./tableHelper');
 
 const setDependencies = ({ lodash }) => (_ = lodash);
 
-const getColumnNames = (collectionRefsDefinitionsMap, columns) => {
+const getColumnNames = ({ columnsDefinitions, isParentActivated = true }) => {
+	const firstActiveIndex = columnsDefinitions.findIndex(item => item.isActivated);
+
+	return columnsDefinitions.map(({ name, isActivated }, index) => {
+		const statement = `${index === 0 || firstActiveIndex === index ? '' : ','} ${name}`;
```

Comment:
I didn't get it. If I have a deactivated column `myColumn` will it look like this: `/*, myColumn*/` ?

Reply:
Yes. The idea was to have a valid syntax once it's uncommented. Also, I used /* */ instead of -- because this plugin does not have a formating